.docker/docker-compose.yml

```diff
@@ -24,17 +34,51 @@ services:
       replicas: 2 
     depends_on:
       - dbpostgres
+    restart: unless-stopped
       
   dbpostgres:
     image: postgres:15.2
     environment:
         POSTGRES_PASSWORD: "${POSTGRES_PASSWORD:-postgres}"
         POSTGRES_USER: "${POSTGRES_USER:-postgres}"
-        POSTGRES_DB: "signs_of_life"
+        POSTGRES_DB: "signs_of_life_crawler"
     expose:
         - "${POSTGRES_PORT}:5432"
     ports:
         - "${POSTGRES_EXTERNAL_PORT:-54320}:5432"
     volumes:
         - ./docker_volumes/postgresdb/data:/var/lib/postgresql/data
         - ./main-db/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+    restart: unless-stopped
+# Todo- Optional - grafana visualisation with the postgress db
+  # grafana-app:
+  grafana:
+    image: grafana/grafana-oss
+    user: "0"
+    environment:
+     GF_RENDERING_SERVER_URL: http://renderer:8081/render
+      #        GF_RENDERING_CALLBACK_URL: http://10.126.73.100:3000/
+      #        GF_LOG_FILTERS: rendering:debug
+          #    expose:
+          #        - "${POSTGRES_PORT}:5432"
+    env_file:
+     - ./conf/grafana/grafana_env
+    ports:
+        - "3000:3000"
+    volumes:
+        - /etc/localtime:/etc/localtime:ro
+        - /etc/timezone:/etc/timezone:ro
+        - ./docker_volumes/grafana/grafana_lib:/var/lib/grafana
+        - ./docker_volumes/grafana/grafana_plugins:/var/lib/grafana/plugins
+        - ./conf/grafana/grafana.ini:/etc/grafana/grafana.ini
+    restart: unless-stopped
+  # grafana-renderer:
+  renderer:
+    image: grafana/grafana-image-renderer:latest
+    ports:
+        - "8081:8081"
+    restart: unless-stopped
+# Todo- NS Resolver for the crawler
+  # resolver:
+# Todo- Optional - elastic visualisation with the postgress db
+  # elk:
```

Comment:
I think if we are going to add grafana it would be best to add it to a different docker compose profile or just be able to turn it off.